flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -2865,4 +2879,19 @@ void OmpAttributeVisitor::IssueNonConformanceWarning(
     context_.Say(source, "%s"_warn_en_US, warnStr);
   }
 }
+void OmpAttributeVisitor::ResolveNames(const parser::OmpObjectList &objList) {
```

Comment:
I am a bit surprised that we needed to define this function. Was the resolution of names not handled anywhere else?

Reply:
I believe I was not understanding the existing code well. I made use of `ResolveOmpObjList` function.